tools/vim/filetype.vim

```diff
@@ -1,4 +1,5 @@
 augroup filetypedetect
  au! BufRead,BufNewFile in.*           setfiletype lammps
  au! BufRead,BufNewFile *.lmp          setfiletype lammps
+ au! BufRead,BufNewFile in.*           setfiletype lammps
```

Comment:
This added line seems redundant

Reply:
Sorry it was meant to be *.in, I was copying my local changes over to a fork.